.github/labeler.yml

```diff
@@ -241,7 +241,7 @@ team-ios:
       - packages/flutter_tools/**/*ios*
       - packages/flutter_tools/**/*macos*
 
-team-release:
+"infra: release":
   - changed-files:
     - any-glob-to-any-file:
       - docs/releases/**/*
```

Comment:
Should `- docs/releases/**/*` be added under `team-infra` label to match the docs?

Reply:
Sg. Done.